src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/DataRepository.cs

```diff
@@ -182,5 +188,56 @@ public async Task<bool> Delete(DataElement dataElement)
 
             return true;
         }
+
+        /// <summary>
+        /// Sets the blob client to point to app owner blob 
+        /// </summary>
+        /// <param name="org">Name of the application owner</param>
+        public void SetAppBlobClient(string org)
```

Comment:
Would it be more correct to relate the client to the org e.g. OrgBlobClient rather than appBlobClient?

Reply:
The usage of Blobs is an implementation detail and should probably not be part of the methods in the interface.

Reply:
Replaced with a function to get OrgDataContext